security/cmd/istio_ca/main.go

```diff
@@ -64,10 +69,12 @@ type cliOptions struct { // nolint: maligned
 	signingKeyFile  string
 	rootCertFile    string
 
-	selfSignedCA                    bool
-	selfSignedCACertTTL             time.Duration
-	selfSignedRootCertCheckInterval time.Duration
-	enableJitterForRootCertRotator  bool
+	selfSignedCA                            bool
+	selfSignedCACertTTL                     time.Duration
+	selfSignedRootCertCheckInterval         time.Duration
+	selfSignedRootCertMinCheckInterval      time.Duration
```

Comment:
Do we need this? Can this be a constant?

Reply:
We need to change these values in our integration tests and perf tests. The default CACertTTL is 10 years, and check interval is one hour.